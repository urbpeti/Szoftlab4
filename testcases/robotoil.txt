Robot picks up oil
Robot R1 35 -3
Item Oil 32
Step
List_Robots
List_Items