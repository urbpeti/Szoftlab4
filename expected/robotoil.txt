R1, Position: 32.0, Velocity: -3.0, Oily, Alive
Oil 32.0 1
